src/RTG/ChatFilter/Loader.php

```diff
@@ -33,64 +33,41 @@
 use pocketmine\event\player\PlayerChatEvent;
 
 class Loader extends PluginBase implements Listener {
-
-    public $words;
+    public $cfg;
 
     public function onEnable() {
-        $this->getServer()->getPluginManager()->registerEvents($this, $this);
-        $this->words = array();
+        $this->getServer()->getPluginManager()->registerEvents($this, $this)
         $this->cfg = new Config($this->getDataFolder() . "words.txt", Config::ENUM);
-
-        $this->words = $this->cfg->getAll();
-    }
-
-    public function onSave() {
-        $this->cfg = new Config($this->getDataFolder() . "words.txt", Config::ENUM);
-        $this->cfg->setAll($this->words);
-        $this->cfg->save();
     }
```

Comment:
As I mentioned, redefining $this->cfg is unnecessary. Since I'm using $this->cfg entirely and deprecated the usage of $this->words, saving a config is as simple as `$this->cfg->save();`, so I also removed the whole onSave() function anyway for simplicity of code. Of course, it is not necessarily better that I replace onSave() with $this->cfg->save(), but in this context, there is not much difference anyway.